.github/workflows/CI.yml

```diff
@@ -0,0 +1,109 @@
+# Run CI tests
+name: CI
```

Comment:
Should we have a more descriptive name like `um2nc CI`?

Reply:
I'm not sure it is necessary as `um2nc` is our only "product" being checked in the repo.

Any opinion @CodeGat?

Reply:
Heh. I didn't see your previous comment @CodeGat. I'll assume you're happy  with this too and resolved.